app/domain/sac_imports/wso2_people_importer.rb

```diff
@@ -39,12 +40,24 @@ def abo_group
       @abo_group ||= Group::AboTourenPortal.first!
     end
 
+    def navision_import_group
+      @navision_import_group ||= Group::ExterneKontakte.find_by!(name: "Navision Import")
+    end
+
     def process_row(row)
       @output.print("#{row[:navision_id]} (#{row[:email]}):")
-      entry = Wso2PersonEntry.new(row, basic_login_group, abo_group)
+      entry = Wso2PersonEntry.new(row, basic_login_group, abo_group, navision_import_group)
       @output.print(entry.valid? ? " ✅\n" : " ❌ #{entry.error_messages}\n")
       if entry.valid?
         entry.import!
```

Comment:
I thought about handling exceptions here, but since we test the run I actually want to see errors directly if something goes wrong while running the import.